demo/BaseComponent/src/manifest.json

```
@@ -2,7 +2,7 @@
     "_version": "1.21.0",
     "sap.app": {
         "id": "my.lib.sample.base",
-        "type": "library",
+        "type": "application",
```

This should be `component` isn't?``

```suggestion
        "type": "component",
```